extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -395,12 +399,16 @@ export class SchemaCompareDialog {
 			let formModel = this.formBuilder.component();
 			await this.view.initializeModel(formModel);
 
-			if (this.sourceIsDacpac) {
-				await this.sourceDacpacRadioButton.focus();
-			} else if (this.sourceIsDatabase) {
-				await this.sourceDatabaseRadioButton.focus();
-			} else if (this.sourceIsProject) {
-				await this.sourceProjectRadioButton.focus();
+			switch (this.sourceEndpointType) {
+				case (mssql.SchemaCompareEndpointType.Database):
+					await this.sourceDatabaseRadioButton.focus();
+					break;
+				case (mssql.SchemaCompareEndpointType.Dacpac):
+					await this.sourceDacpacRadioButton.focus();
+					break;
+				case (mssql.SchemaCompareEndpointType.Project):
+					await this.sourceProjectRadioButton.focus();
```

Comment:
do the project radio buttons show even if the sql projects extension isn't installed? They should be hidden if sql projects isn't there.